core/src/main/scala/org/apache/spark/eventhubs/EventHubsConf.scala

```diff
@@ -368,6 +374,12 @@ final class EventHubsConf private (private val connectionStr: String)
     self.get(OperationTimeoutKey) map (str => Duration.parse(str))
   }
 
+  /** Internal operation timeout. */
+  private[spark] def internalOperationTimeout: FiniteDuration = {
```

Comment:
can we make this a `lazy val` instead of a `def`?

Reply:
sure, will do.